DarkFlippers/unleashed-firmware
# NFC: Add manually MF Classic with custom UID

# What's new

- Add manually all MF Classic with custom UID

# Verification 

- NFC -> Add Manually -> any MF Classic
- Set custom UID. Save dump
- Check saved dump. UID must be the same as user byte input. BCC for 4byte UID must be correct

# Checklist (For Reviewer)
